<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeRebaseWizard</class>
 <widget class="QWizard" name="MergeRebaseWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rebase/Merge</string>
  </property>
  <property name="windowIcon">
   <iconset theme="gitklient">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current branch:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelCurrentBranch">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Source branch:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxFrom"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonMerge">
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRebase">
        <property name="text">
         <string>Rebase</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSquash">
        <property name="text">
         <string>Squash</string>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LogDetailsWidget" name="logDetails">
        <property name="openLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogDetailsWidget</class>
   <extends>QTextBrowser</extends>
   <header>widgets/logdetailswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonMerge</sender>
   <signal>toggled(bool)</signal>
   <receiver>MergeRebaseWizard</receiver>
   <slot>showHideItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonRebase</sender>
   <signal>toggled(bool)</signal>
   <receiver>MergeRebaseWizard</receiver>
   <slot>showHideItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSquash</sender>
   <signal>toggled(bool)</signal>
   <receiver>MergeRebaseWizard</receiver>
   <slot>showHideItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showHideItems()</slot>
 </slots>
</ui>
